database_schema.md

```diff
@@ -83,6 +84,33 @@ For example: a person has two terms as city council member for D4 then a term as
 Roles can also be tied to committee chairs.
 For example: a council member spends a term on the transportation committee and then spends a term on the finance
 committee.
+```
+role_id: {
+    person_id: str
+    person_name: str
+    title: str
+    body_id: str
+    body_name: str
+    start_date: datetime
+    end_date: datetime
+    seat_id: str
+    external_source_id: int
+    created: datetime
+}
+```
+
+### Seat
+A seat is an electable office on the City Council.
+```
+seat_id: {
+    name: str
+    electoral_area: str
+    electoral_type: str
```

Comment:
I assume this is like "council member" vs "aldermen"

Reply:
`electoral_type` is meant to allow for the difference between a council member for a district vs a council member for a citywide position. For example, in Seattle, Position 1 vs Position 9.

I'm not familiar with what "aldermen" means.  How does it fit in with district vs at-large electroral types?

Reply:
aldermen is basically equivalent to all city position if I recall correctly. So "position 9" would be fine.